lib/Specification.cpp

```diff
@@ -32,8 +32,10 @@ namespace anvill {
 
 SpecificationImpl::~SpecificationImpl(void) {}
 
-SpecificationImpl::SpecificationImpl(std::unique_ptr<const remill::Arch> arch_)
+SpecificationImpl::SpecificationImpl(std::unique_ptr<const remill::Arch> arch_, std::string image_name_, std::uint64_t image_base_)
```

Comment:
Should either take a const ref or do `std::move` below. Otherwise two copies happen here.

Reply:
[ccbac6a](https://github.com/lifting-bits/anvill/pull/379/commits/ccbac6a4749f767dabadc244e8739256f5c5e46a)